communication/vehicleconnections/mavsdkstation.cpp

```diff
@@ -80,3 +85,25 @@ QList<QSharedPointer<MavsdkVehicleConnection>> MavsdkStation::getVehicleConnecti
 {
     return mVehicleConnectionMap.values();
 }
+
+void MavsdkStation::on_timeout()
```

Comment:
onTimeout to be consistent
even better: onHeartbeatTimerTimeout

Reply:
I'm following the naming convention used in e.g. mainwindow.cpp in ControlTower where the corresponding slot's name equals the signal name with a "on_" added in the beginning. The signal name "timeout()" is provided by QTimer itself.